calling-apis/problem-set-requests-with-js.md

```diff
@@ -0,0 +1,296 @@
+# Problem Set: Requests With JS
+
+## Directions
+
+Read the problem prompt, which will ask you to write JavaScript code to find certain answers.
+
+Once you found the answers, input them into the answers prompt below.
+
+## Preparation: LocationIQ Forward Geocoding API
+
+Use the [LocationIQ's Forward Geocoding API](https://locationiq.com/geocoding). Before starting this problem prompt:
+
+1. Obtain an API key for LocationIQ's Forward Geocoding API
+1. Familiarize yourself with the LocationIQ API Documentation
+1. Familiarize yourself with the `axios` library
+1. Open Postman for manual testing
+
+![Screenshot of LocationIQ's feature on the Geocoding API](../assets/calling-apis_problem-set-requests-with-js_location-iq-geocoding.png)
+
+## Problem Prompt
+
+Create a JavaScript program that finds the latitude and longitude coordinates for the [seven wonders of the New7Wonders of the World](https://en.wikipedia.org/wiki/New7Wonders_of_the_World), which are the following:
+
+| Wonder              |
+| ------------------- |
+| Great Wall of China |
+| Petra               |
```

Comment:
I've frequently getting back an Austrian hairdresser shop called Petra when the query is just "Petra". Adding in Jordan helps avoid this from happening, but other locations also give strange results some times, so I'm not sure this is worth changing...

Reply:
Seems like we'll have to rethink autograding on Seven Wonders if it's inconsistent...